content/docs/languages/go/quickstart.md

```diff
@@ -39,7 +40,7 @@ The example code is part of the [grpc-go][] repo.
     the repo:
 
     ```sh
-    $ git clone -b {{< param grpc_go_release_tag >}} https://github.com/grpc/grpc-go
+    $ git clone https://github.com/grpc/grpc-go
```

Comment:
We need the latest `helloworld` from **HEAD**. The latest release (v.1.32.0) isn't compatible, it would seem, with `grpc/cmd/protoc-gen-go-grpc@v1.0` -- cf. https://github.com/grpc/grpc-go/pull/3911/files#diff-599c735c60f513e495f0f7065b6e8de6

I'll revert this change once the updates made to `helloworld` are released -- #437 is a reminder.

Reply:
SG. The next release will be October 20th.